fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml

```diff
@@ -16,10 +16,14 @@
             </xpath>
             <xpath expr="//sheet/group[1]" position="before">
                 <div class="oe_button_box" name="button_box"
-                    attrs="{'invisible': [('active', '=', True)]}">
-                    <button class="oe_stat_button bg-danger" type="object"
-                        name="toggle_active" icon="fa-archive">
-                        Archived
+                    >
+                    <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
+                        <!--
+                            this option doesn't works, why?
+                            <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
+                            -->
+                        <label attrs="{'invisible': [('active', '!=', True)]}">Active</label>
+                        <label attrs="{'invisible': [('active', '!=', False)]}">Archived</label>
                     </button>
```

Comment:
```suggestion
                    <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
                        <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
                    </button>
```

Reply:
Also remember remove the line https://github.com/OCA/fleet/blob/5ebf8d3c69a34a1d06f0b2f90155ad903506e217/fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml#L31

Reply:
@mymage can you check @mamcode 's suggestion out?

It should be like this:

```xml
<button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
    <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
</button>
 ```

Reply:
Thanks guys: so the problem was `<field name="active" invisible="1" />`.
Locally tested and works.